app/components/TopNav.vue

```diff
@@ -208,6 +217,24 @@
   left: -2px;
 }
 
+.badge {
+  .radius();
+  font-size: 12px;
+  margin: 0 0 0 8px;
+  padding: 0 4px;
+  text-transform: capitalize;
+  vertical-align: text-bottom;
+  border: 1px solid transparent;
+  line-height: 16px;
+  font-weight: 500;
+  display: inline-block;
+}
+
+.badge--new {
+  background: rgba(135, 54, 224, 0.08);
+  color: #8736e0;
```

Comment:
if these are new colors i think they should probably be defined in `_colors.less` and referenced as variables here